### PR TITLE
[JENKINS-68642] Agents log link is shown to users without permission.

### PR DESCRIPTION
**Pull request by Langer0416**
See [JENKINS-68642](https://issues.jenkins.io/browse/JENKINS-68642).

### Proposed changelog entries

* Show hyperlink to console log of launching node to users with Overall/Administer permission only.

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [X] (If applicable) Jira issue is well described
- [X] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change) and are in the imperative mood. [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [X] Appropriate autotests or explanation to why this change has no tests
- [ ] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadoc, as appropriate. 
- [ ] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@mention

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are accurate, human-readable, and in the imperative mood
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins.io/issues/?filter=12146)).

<a href="https://gitpod.io/#https://github.com/jenkinsci/jenkins/pull/6677"></a>

### CODE REVIEW

File: core/src/main/resources/hudson/slaves/OfflineCause/LaunchFailed/cause.jelly [hunk 1]
```diff
@@ -24,5 +24,9 @@ THE SOFTWARE.
 
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
-  <p class="error">${it} <a href="log">${%See log for more details}</a></p>
+  <p class="error">${it}
```
Comment (Langer0416) [inline core/src/main/resources/hudson/slaves/OfflineCause/LaunchFailed/cause.jelly:27]: I am not sure if it's necessary to add the permission here, here could see the reason why the node was failed.

**Review by daniel-beck — CHANGES_REQUESTED** [review]
Admin permission is the wrong permission to check for according to https://github.com/jenkinsci/jenkins/blob/1d63adb1ce5bb56e8d06b004a5c78c432efe42e6/core/src/main/resources/hudson/slaves/SlaveComputer/log.jelly#L27

**Comment by Langer0416** [comment]
> Did you test it?

Yes, but maybe my test was wrong then, apologizes.

User with permission:
User with permission

User without permission:
User without permission

**Comment by Langer0416** [comment]
Closing it for personal reason.

I have created another pr (See [PR-6734](https://github.com/jenkinsci/jenkins/pull/6734).).

**Pull request closed without merging**